src/ucs/sys/sock.c

```diff
@@ -173,6 +182,25 @@ ucs_status_t ucs_socket_connect(int fd, const struct sockaddr *dest_addr)
     return status;
 }
 
+ucs_status_t ucs_socket_accept(int fd, struct sockaddr *addr, socklen_t *length_ptr,
+                               int *accept_fd)
+{
+    ucs_status_t status;
+
+    *accept_fd = accept(fd, addr, length_ptr);
+    if (*accept_fd == -1) {
```

Comment:
let's use `< 0` for comparison as we did in this source file for other socket functions and below for `bind`/`listen`
wdyt?